bullet_train-obfuscates_id/Gemfile.lock

```diff
@@ -4,6 +4,7 @@ PATH
     bullet_train-obfuscates_id (1.19.2)
       hashids
```

Comment:
Running `bundle install` should have removed this `hashids` line. The fact that it's still here is causing test failures. Can you remove it, please?